tests/console/firewall_enabled.pm

```diff
@@ -16,30 +16,26 @@ use strict;
 use warnings;
 use testapi;
 use version_utils qw(is_upgrade is_jeos is_sle is_vmware is_leap is_tumbleweed);
+use serial_terminal 'select_serial_terminal';
 use Utils::Architectures;
 
 sub run {
     my ($self) = @_;
     if ($self->firewall eq 'firewalld') {
-        my $timeout = 30;
-        $timeout = 60 if is_ppc64le;
-        my $ret = script_run('firewall-cmd --state', timeout => $timeout);
```

Comment:
Has this hide the `running` output before?

Reply:
AFAICS it was not being checked. Only the return code.